frameworks/kafka/README.md

```diff
@@ -1,14 +1,350 @@
+1. Visit http://yourcluster.com/ to view the DC/OS Dashboard.
+1. Navigate to `Universe` => `Packages` and find the `kafka` package.
+1. Click `Install`, then in the pop up dialog click `Advanced` to see the customization dialog.
+1. Make your changes to the default configuration in the customization dialog, then click `Review`.
+1. Examine the configuration summary for any needed changes. Click `Back` to make changes, or `Install` to confirm the settings and install the service.
+
+## Service Settings
+
+### Service Name
+
+Each instance of Apache Kafka in a given DC/OS cluster must be configured with a different service name. You can configure the service name in the **service** section of the install settings. The default service name (used in many examples here) is `kafka`.
+
+All dcos kafka CLI commands have a --name argument allowing the user to specify which instance to query. If you do not specify a service name, the CLI assumes the default value, kafka. The default value for --name can be customized via the DC/OS CLI configuration:
+```
+dcos kafka --name kafka-dev <cmd>
+```
+
+Alternatively, you can install Kafka from the DC/OS web interface. If you install Kafka from the web interface, you must install the Kafka DC/OS CLI subcommands separately. From the DC/OS CLI, enter:
+
+```dcos package install kafka --cli
+```
+
+## Broker Settings
+
+The following settings may be adjusted to customize the amount of resources allocated to each broker. Apache Kafka's requirements (http://kafka.apache.org/documentation.html) must be taken into consideration when adjusting these values. Reducing these values below those requirements may result in adverse performance and/or failures while using the service.
+
+Each of the following settings may be customized under the **broker** configuration section.
+
+### Broker Count
+
+Configure the number of brokers running in a given Kafka cluster. The default count at installation is three brokers. This number may be increased, but not decreased, after installation.
+
+### CPU
+
+The amount of CPU allocated to each broker may be customized. A value of `1.0` equates to one full CPU core on a machine. This value may be customized by editing the **cpus** value under the **broker** configuration section. Turning this too low will result in throttled tasks.
+
+### Memory
+
+The amount of RAM allocated to each broker may be customized. This value may be customized by editing the **mem** value (in MB) under the **broker** configuration section. Turning this too low will result in out of memory errors.
+
+
+### Ports
+
+Each port exposed by the service may be customized via the service configuratiton. If you wish to install multiple instances of the service and have them colocate on the same machines, you must ensure that **no** ports are common between those instances. Customizing ports is only needed if you require multiple instances sharing a single machine. This customization is optional otherwise.
```

Comment:
do we mean the service configuration section of the UI advanced installation page?